docs/index.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: page
-title: AddressBook Level-3
+title: Intern Buddy
 ---
 
 [![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
 [![codecov](https://codecov.io/gh/se-edu/addressbook-level3/branch/master/graph/badge.svg)](https://codecov.io/gh/se-edu/addressbook-level3)
```

Comment:
The link can be updated with this [link](https://codecov.io/gh/AY2223S2-CS2103T-T14-3/tp) instead.